ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -774,4 +775,8 @@ - (NSString * _Nonnull)stringForLoginType:(IDmeWebVerifyLoginType)type {
     }
 }
 
+- (NSString * _Nonnull)urlStringWithURL:(NSString* _Nonnull)url {
```

Comment:
I got confused with this name. I thought it expected a URL but instead it expects a "path string + query string". Could you please rename it?